src/components/app/SingleStrategy/StrategyTokens.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import { Typography } from '@mui/material';
+import EtherScanLink from '../../common/EtherScanLink';
+import TokenCard from './TokenCard';
+import { Network, Strategy } from '../../../types';
+import { getEnv } from '../../../utils/env';
+import CircularProgress from '@mui/material/CircularProgress';
+
+const StyledTypography = styled(Typography)`
+    && {
+        color: ${({ theme }) => theme.title};
+        margin-top: 20px;
+        margin-bottom: 20px;
+        text-align: center;
+    }
+`;
+
+type StrategyTokensProps = {
+    strategy: Strategy;
+    network: Network;
+};
+
+type Token = {
+    balance: number;
+    tokenInfo: {
+        name: string;
+        symbol: string;
+        address: string;
+    };
+};
+
+export const StrategyTokens = (props: StrategyTokensProps) => {
+    const { strategy, network } = props;
+    const [tokensData, setTokensData] = useState<Array<Token>>([]);
+    const { ethplorerKey } = getEnv();
+    // fetching strategy tokens
+    useEffect(() => {
+        fetch(
+            `https://api.ethplorer.io/getAddressInfo/${strategy.token.address}?apiKey=${ethplorerKey}`
+        )
+            .then((response) => response.json())
+            .then((res) => setTokensData(res.tokens));
+    }, []);
+    const data = [];
+    for (const token in tokensData) {
+        data.push({
+            name: tokensData[token].tokenInfo.name,
+            symbol: tokensData[token].tokenInfo.symbol,
+            balance: tokensData[token].balance,
```

Comment:
This balance also likely needs to be formatted since the values being displayed are unreasonably big. eg) right now if you look at the strategy [0x79Be3edA1d7FE92c18B9539bce2611bf8dc9BFc5](https://yearn-watch-fibnhz5we-y-finance.vercel.app/network/ethereum/vault/0x6d765CbE5bC922694afE112C140b8878b9FB0390/strategy/0x79Be3edA1d7FE92c18B9539bce2611bf8dc9BFc5) which is fetching the tokens in the SUSHI contract by mistake, there should only be 3517 USD according to Etherscan (https://etherscan.io/address/0x6B3595068778DD592e39A122f4f5a5cF09C90fE2), not 3517736758

Reply:
But if you look at a different token like FOX, there should only be 200, not 200000000000000000000. It is weird that the formatting is different for those tokens though, maybe the API returns some data on how to normalize it properly